<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Register</class>
 <widget class="QWidget" name="Register">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>843</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="registerlabel">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>40</y>
     <width>201</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Chalkboard SE</family>
     <pointsize>40</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Register</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="input_tel">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>160</y>
     <width>244</width>
     <height>39</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="tel">
   <property name="geometry">
    <rect>
     <x>181</x>
     <y>171</y>
     <width>128</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Phone number:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="input_pw">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>250</y>
     <width>244</width>
     <height>39</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="pw">
   <property name="geometry">
    <rect>
     <x>181</x>
     <y>257</y>
     <width>86</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Password:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="input_user">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>340</y>
     <width>244</width>
     <height>39</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="user">
   <property name="geometry">
    <rect>
     <x>181</x>
     <y>342</y>
     <width>91</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Username:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="clear">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>440</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
  </widget>
  <widget class="QPushButton" name="signup">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>440</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Register</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
